src/views/RoomView.vue

```diff
@@ -80,13 +80,26 @@
             :key="student.id"
             class="col-lg-6"
           >
-            <div class="card my-1">
-              <!--              <img-->
-              <!--                src="https://placehold.co/600x400.png?text=Kamera+Bild"-->
-              <!--                alt=""-->
-              <!--                class="card-img-top"-->
-              <!--              />-->
-              <Camera :user-id="student.id" />
+            <div class="card my-1 d-flex flex-column">
+              <div class="position-relative flex-grow-1">
+                <!-- Die Kamera des Schülers -->
+                <div class="ratio ratio-4x3">
+                  <Camera :user-id="student.id" class="w-100 h-100" />
+                </div>
+
+                <span v-if="channel.isTeacher(channel.currentUser())">
+                  <!-- Der Button oben rechts -->
+                <button class="btn btn-primary position-absolute top-0 end-0"
+                        data-bs-toggle="modal"
+                        data-bs-target="#dropdown-modal"
+                >
+                  <i class="fa fa-ellipsis-v"></i>
+                </button>
+                <DropdownModal :student-id ="student.id" :channel="channel"/>
```

Comment:
hier den `channel` nicht über props übergeben sondern in der Komponente manuell hohlen, sowie in einem anderen Kommentar schon beschrieben